src/target_finder.rs

```diff
@@ -41,7 +41,12 @@ struct Info {
 /// compilation of a test executable, the path will be extracted. Otherwise, it returns `None`.
 fn parse_rustc_command_line(line: &str) -> Option<PathBuf> {
     let trimmed_line = line.trim_start();
-    if !trimmed_line.starts_with("Running `rustc ") {
+    if !trimmed_line.starts_with(&format!(
+        "Running `{}rustc",
+        std::env::var("RUSTC_WRAPPER")
+            .map(|x| format!("{} ", x))
+            .unwrap_or(String::from(""))
```

Comment:
could you please put this in a `lazy_static` or something since it is evaluated for every line.

also, you could use `unwrap_or_default()` instead of `unwrap_or(String::from(""))`.